python/Ganga/Core/GangaRepository/Registry.py

```diff
@@ -323,17 +288,6 @@ def trackandRelease(self):
 
                  flush_thread = threading.Thread(target=self._flush, args=())
                  flush_thread.run()
-
-                 for obj in self.dirty_objs.itervalues():
-
-                    if self.shouldReleaseRun is False:
-                        break
-
-                    _args = ([obj])
-                    release_thread = threading.Thread(target=self._release_lock, args=_args)
```

Comment:
The session file (when working) allows for multiple ganga sessions to ask what is locked/unlocked and to operate on job objects that were previously locked.
I know this is a minor use case and it often leads to repository corruptions but these corruptions are more to do with bad use of the locks rather than the principle being broken.

I can add the equivalent line into the prep repo in a PR to allow for the repo to be released after each operation, as it's only the prep repo this performance hit is relatively minor.

I've had briefly 2 or more monitoring threads on different systems operating on multiple different job objects without too many problems. (Ganga shouts loudly when it can't get access to a job for monitoring but eventually moves onto the next job).